substrate/frame/examples/basic/src/lib.rs

```diff
@@ -377,46 +416,31 @@ pub mod pallet {
 	// Any storage must be one of `StorageValue`, `StorageMap` or `StorageDoubleMap`.
 	// The first generic holds the prefix to use and is generated by the macro.
 	// The query kind is either `OptionQuery` (the default) or `ValueQuery`.
-	// - for `type Foo<T> = StorageValue<_, u32, OptionQuery>`:
-	//   - `Foo::put(1); Foo::get()` returns `Some(1)`;
-	//   - `Foo::kill(); Foo::get()` returns `None`.
-	// - for `type Foo<T> = StorageValue<_, u32, ValueQuery>`:
-	//   - `Foo::put(1); Foo::get()` returns `1`;
-	//   - `Foo::kill(); Foo::get()` returns `0` (u32::default()).
+	// - for `type Dummy<T> = StorageValue<_, u32, OptionQuery>`:
+	//   - `Dummy::put(1); Foo::get()` returns `Some(1)`;
+	//   - `Dummy::kill(); Foo::get()` returns `None`.
+	// - for `type Dummy<T> = StorageValue<_, u32, ValueQuery>`:
+	//   - `Dummy::put(1); Foo::get()` returns `1`;
+	//   - `Dummy::kill(); Foo::get()` returns `0` (u32::default()).
 	#[pallet::storage]
-	pub(super) type Dummy<T: Config> = StorageValue<_, T::Balance>;
```

Comment:
Instead of deleting them, please do something interesting with it as an example.  

As this is an example pallet: it should showcase some sensible things that can be done with storage items etc. And it should document these things so that a beginner can understand it.